UI/GameSettingsScreen.cpp

```diff
@@ -833,7 +833,7 @@ void GameSettingsScreen::CreateViews() {
 	qc5->SetEnabledPtr(&g_Config.bEnableQuickChat);
 #endif
 
-	networkingSettings->Add(new ItemHeader(n->T("Misc", "Misc (default = compatibility)")));
```

Comment:
Instead of changing the Section Header, you should change the one specific to Port Offset (ie. "Port offset (0 = PSP compatibility)" ), because there are more options in Misc that most common people shouldn't touch when not having any issue.

Reply:
oops sry.

Reply:
Nevermind, i got confuse too, by saying "default = PSP compatibility" doesn't right either when default port offset is non-zero ... i wonder what kind of wording we should use...

Reply:
May be we should just replace it with "Misc (workaround)" ? since these options are workaround anyway LOL

Reply:
> Nevermind, i got confuse too, by saying "default = PSP compatibility" doesn't right either when default port offset is non-zero ... i wonder what kind of wording we should use...

btw where is that?
Like I am trying to find it in the code but its not in the GameSettingsScreen.cpp (which it should be?) lol.

Reply:
Sorry, i mean the "Misc (default = compatibility)" one